content/guides/integration-guides/csi-integration/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: CSI Integration Guides
+menuTitle: Kubernetes-CSI
+description: "All the Container Storage Interface(CSI) Integration guides for different Soda CSI solutions"
+weight: 1
+disableToc: true
+---
+
+{{%children style="h3" description="true" %}}  
```

Comment:
Page looks like this for me, pls have a look
![image](https://user-images.githubusercontent.com/30930862/110905644-cd3e1980-8330-11eb-8acf-f59182ae047f.png)

Reply:
this is will get rendered on the website properly https://docs.sodafoundation.io/guides/integration-guides/csi-integration/